src/main/java/com/basho/riak/presto/CoverageRecordCursor.java

```diff
@@ -39,12 +38,14 @@
 
     private final String schemaName;
     private final String tableName;
+    private final String bucketName;
```

Comment:
To define several presto table corresponding to one riak bucket, `bucketName` is needed aside from `tableName`.

For example, there is a JSON data in the riak bucket named `ec (e-commerce)` as follows.
On presto side, this `ec bucket` is expressed as two diffrent table like `customer table` and `cart table`.

```
ec bucket

{
	"name":"Taka",
	"place":"JP",
	"cart":[
		{"name":"beer","category":"drink","price":200,"num":5},
		{"name":"chicken","category":"food","price":300,"num":1},
		{"name":"dictionary","category":"book","price":1000,"num":1}
	]
}
```

To define presto table, we have to describe table definition in JSON.
The following example means the `customer table` correspond to the riak bucket named `customer`, and the `cart table` correspond to the riak bucket named `cart`.
But in this case, both customer data ard cart data are stored in the `ec bucket` as one nested JSON.

```
customer table definition

{
	"name" : "customer",
	"columns":[
		{"name":"name", "type":"varchar", "index":true},
		{"name":"place", "type":"varchar", "index":true}
	]
}

cart table definition

{
	"name":"cart",
	"columns":[
		{"name":"name",type":"varchar","index":true},
		{"name":"category","type":"varchar","index":true},
		{"name":"price","type":"bigint","index":true},
		{"name":"num","type":"bigint","index":true}
	]
}
```

Therefore, I added `bucketName` field aside from `tableName`